sdk/storage/Azure.ResourceManager.Storage/tests/Helpers/StorageTestBase.cs

```diff
@@ -11,6 +11,8 @@
 using NUnit.Framework;
 using Azure.Core;
 using Azure.ResourceManager.Models;
+using Castle.DynamicProxy;
```

Comment:
Looks like you're changing a file in the storage SDK. Any particular reason? I think we need to revert the changes to this file.

Reply:
This one accidentally slipped in, reverting the change now

Reply:
Just reverted it back to normal